schema/url/sequence.raml

```diff
@@ -0,0 +1,22 @@
+#%RAML 1.0 Library
+
+
+uses:
+  common: ../common.raml
+
+types:
+  SequencePostRequest:
+    additionalProperties: false
+    properties:
+      features:
+        type: array
+        items: common.Feature
+      deflineFormat: common.DeflineFormat
+      forceStrandedness:
+        type: boolean
+        default: false
+      basesPerLine:
+        type: integer
+        default: 60
```

Comment:
Curious if these defaults work?  Like if you omit it does the value get set in the generated Java object?

Reply:
Yeah I think at least some of the defaults do, from what I remember!

Reply:
Checked and yes, they do work!